src/field_10x26_impl.h

```diff
@@ -454,18 +454,15 @@ void secp256k1_fe_sqr_inner(uint32_t *r, const uint32_t *a);
 
 #else
 
-#ifdef VERIFY
```

Comment:
Also remove these from src/field_5x52_int128_impl.h?

Reply:
fixed and also updated some of comment